app/controllers/saasbook_controller.rb

```diff
@@ -2,13 +2,18 @@
 
 # Controller for the Saasbook website. Temporary maybe?
```

Comment:
can we remove the temporary maybe portion since it is now required

Reply:
This is part of a [separate pivotal task](https://www.pivotaltracker.com/story/show/181976893). 

Generally a PR will focus only on the tasks directly related to it; chores like comment cleanup should be handled separately.